rules-java/tests/src/test/resources/classmetadatatest/sampleclasses/com/rhc/booking/services/EventServer.java

```diff
@@ -1,8 +0,0 @@
-package com.rhc.booking.services;
```

Comment:
There is another same source file at src/test/resources/classmetadatatest/src/com/rhc/booking/services/EventServer.java I was confused by duplicated files